src/utils/accountUtils.ts

```diff
@@ -8,6 +8,17 @@ import BigNumber from 'bignumber.js'
 
 export type AccountNetTypeOpt = 'all' | NetType.NetType
 
+export const solAddressToHelium = (solanaAddress: string) => {
```

Comment:
I think we can remove these functions now, they're in `spl-utils`
`import { heliumAddressToSolAddress, heliumAddressFromSolAddress } from '@helium/spl-utils'`